prog/test/hetzner_server.rb

```diff
@@ -99,7 +99,21 @@ def verify_specs_installation(installed: true)
     vm_host.sshable.cmd("sudo chmod a+rw #{tmp_dir}")
     vm_host.sshable.cmd("sudo RUN_E2E_TESTS=1 SPDK_TESTS_TMP_DIR=#{tmp_dir} bundle exec rspec host/e2e")
 
-    hop_wait
+    hop_download_boot_images
+  end
+
+  label def download_boot_images
```

Comment:
`Prog::Vm::HostNexus.assemble` has `default_boot_images` parameter, if you pass it, it will download images automatically.

Reply:
:ack: updated this